src/components/Checkout/DonateRepaymentsToggle.vue

```diff
@@ -49,6 +93,8 @@ export default {
 			myDonateRepayments: false,
 			totals: {},
 			loans: [],
+			isLightboxVisible: false,
+			checkboxRight: true,
```

Comment:
What does this boolean help you with?

Reply:
isLightboxVisible - Allows me to open and close the newly included lightbox.
checkboxRight - is swapping the display of the label and input.

Reply:
this checkboxRight boolean allows me to toggle between a checkbox with a label on the right and a label on the right. Going to refactor this now and just always set it to right.